### ISSUE TITLE
Error Prone mode: always suggest `@Nullable Void`

### ISSUE THREAD

**Issue by aosen-xiong** [eisop/checker-framework#2101]
Under `eisopcf`, a `Void` type argument in `@NullMarked` code is reported, and the only fix offered is `@SuppressWarnings("eisopcf")`. This is most of what the plugin reports on real code: in reactor-pool, 39 of the Checker Framework's 44 findings are `Mono<Void>`.

```java
import org.jspecify.annotations.NullMarked;

@NullMarked
class Pool {
    static class Mono<T> {}

    void release(Mono<Void> completion) {}
}
```

```
$ javac -cp jspecify-1.0.1.jar -processor nullness Pool.java
Pool.java:7: error: [type.argument.type.incompatible] incompatible type argument for type parameter T of Mono.
    void release(Mono<Void> completion) {}
                      ^
  found   : @Nullable Void
  required: @NonNull Object
1 error
```

**Expected:** for every `Void` written without `@Nullable`, the Error Prone plugin suggests `@Nullable Void` and imports `org.jspecify.annotations.Nullable`:

```java
void release(Mono<@Nullable Void> completion) {}
```

**Environment:** EISOP Checker Framework 3.49.5-eisop2-SNAPSHOT, JDK 21. Found when running the Error Prone plugin on reactor-pool.
